docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.icons.scss

```diff
@@ -22,6 +22,12 @@
     &:focus::before {
       background: svg('<svg viewBox="0 0 12 12"><path d="M11.273 5.818v5.455H.727V.727h5.091m1.455 0h4v4m0-4L6.182 5.818" fill="transparent" stroke="#{$hb-color--primary}" stroke-width="2" stroke-miterlimit="10"/></svg>') center center no-repeat;
     }
+
+    .hb-local-footer--dark & {
+      &::before {
```

Comment:
I think you need the :Focus/:hover :before settings like above for the hover styles to apply properly!